doc/sphinx-guides/source/installation/shibboleth.rst

```diff
@@ -39,28 +39,10 @@ Install Shibboleth
 
 Installing Shibboleth will give us both the ``shibd`` service and the ``mod_shib`` Apache module.
 
-Enable Shibboleth Yum Repo
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-This yum repo is recommended at https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPLinuxRPMInstall
-
-``cd /etc/yum.repos.d``
-
-Install ``wget`` if you don't have it already:
-
-``yum install wget``
-
-If you are running el8 (RHEL/derivative 8):
-
-``wget http://download.opensuse.org/repositories/security:/shibboleth/CentOS_8/security:shibboleth.repo``
-
-If you are running el7 (RHEL/CentOS 7):
-
-``wget http://download.opensuse.org/repositories/security:/shibboleth/CentOS_7/security:shibboleth.repo``
-
-If you are running el6 (RHEL/CentOS 6):
+Install Shibboleth Yum Repo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``wget http://download.opensuse.org/repositories/security:/shibboleth/CentOS_CentOS-6/security:shibboleth.repo``
+The Shibboleth Project now provides `a web form <https://shibboleth.net/downloads/service-provider/RPMS/>` to generate an appropriate package repository for use with YUM/DNF.
```

Comment:
Consistency and fix link.

```suggestion
The Shibboleth project now provides `a web form <https://shibboleth.net/downloads/service-provider/RPMS/>`_ to generate an appropriate package repository for use with YUM/DNF.
```

Reply:
The web form gives me something like this for CentOS 8:

```
[shibboleth]
name=Shibboleth (CentOS_8)
# Please report any problems to https://shibboleth.atlassian.net/jira
type=rpm-md
mirrorlist=https://shibboleth.net/cgi-bin/mirrorlist.cgi/CentOS_8
gpgcheck=1
gpgkey=https://shibboleth.net/downloads/service-provider/RPMS/repomd.xml.key
        https://shibboleth.net/downloads/service-provider/RPMS/cantor.repomd.xml.key
enabled=1
```

Will people reading this page know to put this in  `cd /etc/yum.repos.d`? The previous instructions were pretty explicit about this.